APKBUILD

```
@@ -13,9 +13,10 @@ nsswitch.conf
 ld.so.conf"
 subpackages="$pkgname-bin $pkgname-dev $pkgname-i18n"
 triggers="$pkgname-bin.trigger=/lib:/usr/lib:/usr/glibc-compat/lib"
+options="lib64"
 
 package() {
```

Suggest adding this line to prevent issues for users of `libc6-compat`.
```suggestion
package() {
  conflicts="libc6-compat"
```

---

To illustrate this problem in action:
```
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/main/x86_64/APKINDEX.tar.gz
fetch https://dl-cdn.alpinelinux.org/alpine/v3.15/community/x86_64/APKINDEX.tar.gz
(1/10) Installing glibc (2.35-r0)
(2/10) Installing ncurses-terminfo-base (6.3_p20211120-r1)
(3/10) Installing ncurses-libs (6.3_p20211120-r1)
(4/10) Installing readline (8.1.1-r0)
(5/10) Installing bash (5.1.16-r0)
Executing bash-5.1.16-r0.post-install
(6/10) Installing libc6-compat (1.2.2-r7)
ERROR: libc6-compat-1.2.2-r7: trying to overwrite lib64/ld-linux-x86-64.so.2 owned by glibc-2.35-r0.
(7/10) Installing libgcc (10.3.1_git20211027-r0)
(8/10) Installing glibc-bin (2.35-r0)
(9/10) Installing glibc-dev (2.35-r0)
(10/10) Installing glibc-i18n (2.35-r0)
Executing busybox-1.34.1-r7.trigger
Executing glibc-bin-2.35-r0.trigger
1 error; 109 MiB in 24 packages

Exited with code exit status 1
```
([source](https://app.circleci.com/pipelines/github/sgerrand/alpine-pkg-glibc/32/workflows/9c5ba9ea-1a5b-47aa-b684-eeef8bb09c74/jobs/296))

---

I see why this change is probably needed, but doesn't this conflict with https://github.com/sgerrand/alpine-pkg-glibc/blob/0d226ba64a85574fd22ddc626fda2a51af18e18b/APKBUILD#L33-L34 which actually installs `libc6-compat` as a dependency of glibc-bin? 

Is it possible that dependency added in https://github.com/sgerrand/alpine-pkg-glibc/commit/39878629cbf803bc70323f35dfeb674da9a5c0ac be removed? It's a bit confusing as to why if we are installing glibc that we would want a dependency on a musl/glibc compatibility layer like libc6-compat as it does in the `2.35-r0`/`2.35-r1` glibc-bin packages and not so clear which user-reported issues were being addressed by that addition.